src/proxy_handler.cc

```diff
@@ -128,7 +202,36 @@ ProxyHandler::HandleRequest(const Request& request, Response* response)
       header.pop_back();
 	  std::string head = header.substr(0, header.find(":"));
 	  std::string value = header.substr(header.find(":") + 2);
-	  response->AddHeader(head, value);
+
+	  if(head == "Cache-Control" 
+	     && value.find("no-store") == std::string::npos 
```

Comment:
That would be great. I can work on it later in the day too if you need any help. :)